sql/core/src/test/resources/sql-tests/inputs/pgSQL/case.sql

```diff
@@ -0,0 +1,267 @@
+--
+-- Portions Copyright (c) 1996-2019, PostgreSQL Global Development Group
+--
+--
+-- CASE
+-- https://github.com/postgres/postgres/blob/REL_12_BETA1/src/test/regress/sql/aggregates.sql
+-- Test the case statement
+--
+-- There are 2 join condition is missing in this test case. we set spark.sql.crossJoin.enabled=true.
+set spark.sql.crossJoin.enabled=true;
```

Comment:
Need to `set spark.sql.crossJoin.enabled=true`. otherwise:
```
-- !query 30
SELECT '' AS Five, NULLIF(a.i,b.i) AS `NULLIF(a.i,b.i)`,
  NULLIF(b.i, 4) AS `NULLIF(b.i,4)`
  FROM CASE_TBL a, CASE2_TBL b
-- !query 30 schema
struct<>
-- !query 30 output
org.apache.spark.sql.AnalysisException
Detected implicit cartesian product for INNER join between logical plans
Project [i#x]
+- Relation[i#x,f#x] parquet
and
Project [i#x]
+- Relation[i#x,j#x] parquet
Join condition is missing or trivial.
Either: use the CROSS JOIN syntax to allow cartesian products between these
relations, or: enable implicit cartesian products by setting the configuration
variable spark.sql.crossJoin.enabled=true;
```